hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.js

```diff
@@ -74,6 +93,10 @@ const Dashboard = () => {
                     reportIncident={reportIncident}
                 />
                 <ReturnedTable items={itemsReturned} />
+
+                <Button variant="contained" onClick={toggleMenu}>
```

Comment:
Make sure we remove this eventually, but it's fine to keep in here for now until we have it hooked up to redux